web/src/main/java/no/nav/sbl/dialogarena/modiabrukerdialog/web/rest/aaputsending/AAPUtsendingService.kt

```diff
@@ -0,0 +1,106 @@
+package no.nav.sbl.dialogarena.modiabrukerdialog.web.rest.aaputsending
+
+import no.nav.common.leaderelection.LeaderElectionClient
+import no.nav.sbl.dialogarena.modiabrukerdialog.api.domain.gsak.Sak
+import no.nav.sbl.dialogarena.modiabrukerdialog.api.domain.henvendelse.Fritekst
+import no.nav.sbl.dialogarena.modiabrukerdialog.api.domain.henvendelse.Melding
+import no.nav.sbl.dialogarena.modiabrukerdialog.api.domain.henvendelse.Meldingstype
+import no.nav.sbl.dialogarena.modiabrukerdialog.api.service.HenvendelseUtsendingService
+import no.nav.sbl.dialogarena.modiabrukerdialog.api.service.gsak.SakerService
+import no.nav.sbl.dialogarena.modiabrukerdialog.web.rest.dialog.RequestContext
+import no.nav.sbl.dialogarena.modiabrukerdialog.web.rest.dialog.getKanal
+import java.net.InetAddress
+import java.util.*
+import java.util.concurrent.Executors
+import java.util.concurrent.Future
+import java.util.concurrent.atomic.AtomicBoolean
+import java.util.concurrent.atomic.AtomicInteger
+
+const val RPA_IDENT = "Z999999"
+const val RPA_ENHET = "2830" //4151
+
+const val MELDING_FRITEKST = """
+Det har dessverre skjedd en teknisk feil som medfører at utbetalingen din bli forsinket.
+Vi beklager dette, og vi jobber nå med å rette opp feilen.
+Pengene vil være inne på din konto i løpet av denne uken.
+"""
+const val MELDING_TILKNYTTETANSATT = false
+const val MELDING_TEMAKODE = "AAP"
+const val MELDING_TEMAGRUPPE = "ARBD"
+
+class AAPUtsendingService(
+        private val sakerService: SakerService,
+        private val henvendelseService: HenvendelseUtsendingService,
+        private val leaderElection: LeaderElectionClient
+) {
+    private val hasStarted = AtomicBoolean(false)
+    private val total = AtomicInteger(0)
+    private val prosessCount = AtomicInteger(0)
+    private val executor = Executors.newSingleThreadExecutor()
+    private var job: Future<*>? = null
```

Comment:
Sikkert i overkant mye `Atomic` her, men bare for å sikre at ting ikke kan starte i parallell